native/src/routes/feedback/containers/FeedbackModalContainer.js

```diff
@@ -1,33 +1,296 @@
+    return new FeedbackVariant({
+      label: this.props.t('technicalTopics'),
+      feedbackType: CATEGORIES_FEEDBACK_TYPE,
+      feedbackCategory: TECHNICAL_FEEDBACK_CATEGORY
+    })
+  }
+
+  getContentFeedbackOption = (): ?FeedbackVariant => {
+    const { navigation, t } = this.props
+    const cityName = this.getCityName()
+
+    if (cityName) {
+      const label = t('contentOfCity', { city: cityName })
+      const feedbackCategory = CONTENT_FEEDBACK_CATEGORY
+      const feedbackType = navigation.getParam('type')
+      switch (feedbackType) {
+        case 'Event':
+          return new FeedbackVariant({ label, feedbackType: EVENTS_FEEDBACK_TYPE, feedbackCategory })
+        case 'Offers':
+          return new FeedbackVariant({ label, feedbackType: OFFERS_FEEDBACK_TYPE, feedbackCategory })
+        default:
+          return new FeedbackVariant({ label, feedbackType: CATEGORIES_FEEDBACK_TYPE, feedbackCategory })
+      }
+    }
+  }
+
+  getOffersFeedbackOptions = (): Array<FeedbackVariant> => {
+    const { t } = this.props
+    const { offers } = this.props.navigation.state.params
+    if (offers) {
+      return offers.map(offer =>
+        new FeedbackVariant({
+          label: t('contentOfOffer', { offer: offer.title }),
+          feedbackType: OFFER_FEEDBACK_TYPE,
+          feedbackCategory: CONTENT_FEEDBACK_CATEGORY,
+          alias: offer.alias
+        })
+      )
+    }
+    return []
+  }
 
-class FeedbackModalContainer extends React.Component<{| navigation: NavigationStackProp<*> |}> {
   closeModal = () => this.props.navigation.goBack()
 
-  sendFeedback = async (feedbackData: FeedbackParamsType) => {
-    const apiUrl = await determineApiUrl()
-    const feedbackEndpoint = createFeedbackEndpoint(apiUrl)
-    await feedbackEndpoint.request(feedbackData)
+  getCurrentPageFeedbackOption = (): ?FeedbackVariant => {
+    const { navigation, t } = this.props
+    const { type, path, title, feedbackAlias, query } = navigation.state.params
+
+    const feedbackCategory = CONTENT_FEEDBACK_CATEGORY
+    if (type === 'Category' && path && title) {
+      return new FeedbackVariant({
+        label: t('contentOfPage', { page: title }),
+        feedbackType: PAGE_FEEDBACK_TYPE,
+        feedbackCategory
+      })
+    } else if (type === 'Pois' && path && title) {
+      return new FeedbackVariant({
+        label: t('contentOfPoi', { poi: title }),
+        feedbackType: PAGE_FEEDBACK_TYPE,
+        feedbackCategory
+      })
+    } else if (type === 'Event' && path && title) {
+      return new FeedbackVariant({
+        label: t('contentOfEvent', { event: title }),
+        feedbackType: PAGE_FEEDBACK_TYPE,
+        feedbackCategory
+      })
+    } else if (type === 'Offers' && feedbackAlias && title) {
+      return new FeedbackVariant({
+        label: t('contentOfOffer', { offer: title }),
+        feedbackType: OFFER_FEEDBACK_TYPE,
+        feedbackCategory
+      })
+    } else if (type === 'Search' && query) {
```

Comment:
Seems that this endpoint is never called. Same goes for web (as this part is just copy pasted). Will remove this.

Reply:
You mean it is never called here, right? There is a separate place handling search feedback, isn't it?

Reply:
Yes, that's what I meant.